src/board.py

```diff
@@ -61,22 +61,23 @@ def place_stone(self, i:int, j:int, color : Color) -> 'Board':
         new_board[i]
 
     """
-    hasCell function checks if the @param cell exists in the board
+    has_cell function checks if the @param cell exists in the board
     @return True iff the cell is in the boudaries of the board
     """
 
-    def hasCell(self, cell: Cell) -> bool:
+    def has_cell(self, cell: Cell) -> bool:
         return False
 
     """
-    findNeighbours function finds all neighbouring cells
+    find_neighbours function finds all neighbouring cells
     in the board to the cell defined by @param coords
     @return list of neighbouring cells
     """
 
-    def findNeighbours(self, coords: tuple) -> list[Cell]:
+    def find_neighbours(self, coords: tuple) -> list[Cell]:
```

Comment:
I think it would make more sense for this to return a set  of cells: a neigboirod is a set as it has no order

Reply:
Also for coords, can we give it the more specific type tuple(int,int)?

Reply:
tuple(int,int) is now the specified type